qiskit_optimization/algorithms/cplex_optimizer.py

```diff
@@ -84,6 +94,19 @@ def disp(self, disp: bool):
         """
         self._disp = disp
 
+    @property
+    def cplex_parameters(self) -> Dict[str, Any]:
+        """Returns parameters for CPLEX"""
+        return self._cplex_parameters
+
+    @cplex_parameters.setter
+    def cplex_parameters(self, parameters: Dict[str, Any]):
```

Comment:
I guess parameters should be the same as in the constructor, `Optional[Dict[str, Any]]` in both getter/setter?

Reply:
Because we cannot use the empty dict `{}` as a default value of the method, I used `Optional`. But, setter/getter does not need to use the default value as parameter. So, I prefer without `Optional`.

Reply:
I see, usually we have same type hints in properties as in the constructor. I'd rather add a check for `None` in the setter anyway if `None` value may cause an error later.

Reply:
It's OK to raise a ValueError if None is given.

Reply:
I checked the manual again and noticed that `cplex_parameters` is optional. So, `None` is treated as no option. So, I change it to accept `None`.